packages/cadl-ranch-specs/http/payload/xml/main.tsp

```diff
@@ -32,6 +34,55 @@ model ModelWithOptionalField {
   value?: int32;
 }
 
+@doc("Contains fields that are XML attributes.")
+model ModelWithAttributes {
+  @attribute id1: int32;
+  @attribute id2: string;
+  enabled: boolean;
+}
+
+@doc("Contains fields of wrapped and unwrapped arrays of primitive types.")
+model ModelWithUnwrappedArray {
+  @unwrapped colors: string[];
+  counts: int32[];
+}
+
+@doc("Contains fields of wrapped and unwrapped arrays of primitive types that have different XML representations.")
+model ModelWithRenamedArrays {
+  @name("Colors") @unwrapped colors: string[];
+  @name("Counts") counts: int32[];
+}
+
+@doc("Contains fields of the same type that have different XML representation.")
```

Comment:
a little confused about "same type that have different XML representation"

Reply:
This type has two fields that are both of type `SimpleModel` but have different XML names, `InputData` and `OutputData`.

It's the same as [this](https://github.com/Azure/azure-rest-api-specs/blob/a58614a53ecd06c6e6638b58c407d5bd8e61c009/specification/storage/data-plane/Microsoft.BlobStorage/stable/2021-12-02/blob.json#L10870-L10880) scenario in storage.